skimage/io/tests/test_pil.py

```diff
@@ -39,6 +39,13 @@ def setup_module(self):
     except ImportError:
         pass
 
+def test_png_round_trip():
+    f = NamedTemporaryFile(suffix='.png')
+    fname = f.name
```

Comment:
I don't think so. When f is garbage collected at the end of the function, the temporary file will be removed per the NamedTemporaryFile spec: "If delete is true (the default), the file is deleted as soon as it is closed."